components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -159,8 +159,22 @@ class Airwallex internal constructor(
                 transactionCurrency = params.transactionCurrency,
                 transactionMode = params.transactionMode,
                 countryCode = params.countryCode
-            ),
-            listener
+            )
+        )
+        val filteredItems = response?.items?.let { items ->
```

Comment:
Is it possible to centralise this filtering and the filtering in the view model?

Reply:
Yeah the only thing is `canHandleSessionAndPaymentMethod` needs a activity as input, and view model doesn't have it.